test/test_helper.rb

```diff
@@ -54,6 +54,17 @@
   ]
 )
 
+class IsolatedTest < Minitest::Test
```

Comment:
A little comment goes a long way :relaxed: 
```suggestion
class IsolatedTest < Minitest::Test
  # It cleans the active_hash database before and after (around) the test. 
  # Needed for tests calling Model::X.create directly to prevent ID errors.
```